src/test/java/fr/esgi/al5_2/Tayarim/ProprietaireControllerTest.java

```diff
@@ -36,279 +37,349 @@
+  private MockMvc mockMvc;
+
+  @MockBean
+  private ProprietaireService proprietaireService;
+
+  @MockBean
+  private AuthService authService;
+
+  @Autowired
+  private ObjectMapper objectMapper;
+  private ProprietaireCreationDTO proprietaireCreationDTO;
+  private Proprietaire proprietaire;
+  private List<ProprietaireDTO> proprietaireDTOS;
+  private ProprietaireDTO proprietaireDTO;
+  private ProprietaireDTO proprietaireDTO2;
+  private ProprietaireDTO proprietaireDTOWithLogement;
+  private ProprietaireUpdateDTO proprietaireUpdateDTO;
+
+  @BeforeEach
+  public void init() {
+    LocalDateTime localDateTime = LocalDateTime.now();
+    proprietaireCreationDTO = ProprietaireCreationDTO.builder()
+        .nom("Ferreira")
+        .prenom("Mathieu")
+        .email("test@gmail.com")
+        .numTel("0612345678")
+        .build();
+    proprietaire = Proprietaire.builder()
+        .nom("Ferreira")
+        .prenom("Mathieu")
+        .email("test@gmail.com")
+        .numTel("0612345678")
+        .motDePasse("$2a$12$3hQDUblvPShmuQg/.g0Qk.wHAGjqPL54RMO/lNgsei/HQGo0ZLIYm")
+        .dateInscription(localDateTime)
+        .isPasswordUpdated(Boolean.TRUE)
+        .build();
+    proprietaireDTO = ProprietaireDTO.builder()
+        .id(1L)
+        .nom("Ferreira")
+        .prenom("Mathieu")
+        .email("test@gmail.com")
+        .numTel("0612345678")
+        .dateInscription(localDateTime)
+        .logements(null)
+        .build();
+    proprietaireDTO2 = ProprietaireDTO.builder()
+        .id(2L)
+        .nom("Jhonny")
+        .prenom("Pasdoe")
+        .email("test2@gmail.com")
+        .numTel("0612345679")
+        .dateInscription(localDateTime)
+        .logements(null)
+        .build();
+    proprietaireDTOS = List.of(proprietaireDTO, proprietaireDTO2);
+    proprietaireDTOWithLogement = ProprietaireDTO.builder()
+        .id(1L)
+        .nom("Ferreira")
+        .prenom("Mathieu")
+        .email("test@gmail.com")
+        .numTel("0612345678")
+        .dateInscription(localDateTime)
+        .logements(List.of(new LogementDTO(1L, 1L)))
+        .build();
+    proprietaireUpdateDTO = ProprietaireUpdateDTO.builder()
+        .prenom("Karl")
+        .build();
+  }
+
+  @Test
+  public void ProprietaireController_CreerProprietaire_ReturnCreated() throws Exception {
+    given(proprietaireService.creerProprietaire(ArgumentMatchers.any())).willAnswer(
+        invocationOnMock -> proprietaireDTO);
+    when(authService.verifyToken(any(), anyBoolean())).thenReturn(
+        new AbstractMap.SimpleEntry<>(proprietaireDTO.getId(), true)); //skip token verif
+
+    ResultActions response = mockMvc.perform(post("/proprietaires")
+        .header("Authorization",
+            "Bearer eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJqb2huZG9lQGdtYWlsLmNvbTs2MjZjMjlhNS0zZGU0LTQ5Y2YtODI4ZS1hNDkxNWQzMzM5N2EiLCJpYXQiOjE3MTQxMTM1NzcsImV4cCI6MTcxNDExNzE3N30.kbU1pVkUHkRXktX44JBFN_xzDv-ZvSmtPjnjORO0vHPiHd3f2MGfDF15VTFO5icIrU_bV9cTqZ70RDlKdlp0-g")
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 261).